lib/matplotlib/backends/backend_pgf.py

```diff
@@ -23,37 +23,55 @@
     _create_pdf_info_dict, _datetime_to_pdf)
 from matplotlib.path import Path
 from matplotlib.figure import Figure
+from matplotlib.font_manager import FontProperties
 from matplotlib._pylab_helpers import Gcf
 
 _log = logging.getLogger(__name__)
 
 
+DOCUMENTCLASS = r"\documentclass{article}"
+
```

Comment:
Agreed that the linting rule is non-obvious here.

However, I guess that adding an additional empty line here should most likely solve it.

Reply:
Hm OK, I didn’t make the connection to that line since it mentioned line 38, but I can try if adding a line there changes anything.